package.json

```diff
@@ -21,7 +33,7 @@
     "stellar"
   ],
   "engines": {
-    "node": ">=4"
+    "node": ">=6.14.0"
```

Comment:
Any reason for `6.14`? I think `>=6` should be good.

Reply:
ESLint, possibly more tools, only support 6.14 and above. Since built files will be ES5ed, they'll still work on 6.0-6.13, but I guess we could miss some bugs that are fixed in 6.14. But I think that's pretty unlikely. 

So the Travis files are set to install 6.14, and that's the version in NPM, so it makes sense to set the engine to the same value, since Babel could in the future use that to determine its transpile target.